pyEpiabm/pyEpiabm/sweep/intervention_sweep.py

```diff
@@ -87,13 +89,16 @@ def __call__(self, time):
                             self._population.cells))
             if intervention.is_active(time, num_cases):
                 if self.intervention_active_status[intervention] is False:
-                    # Update with the single intervention
-                    # (with respect to index)
+                    # If the next same type intervention is activated,
+                    # update the parameter values with this new intervention
+                    # whose values could be retrieved with respect to index
                     if hasattr(intervention, 'occurrence_index'):
+                        # Get the intervention type
                         intervention_type = \
                             list(self.intervention_dict.keys())[list(
                                 self.intervention_dict.values()).index(
                                 type(intervention))]
+                        # Update parameter values
```

Comment:
```suggestion
                        # Update parameter values with current active intervention
```

Would this be an accurate description of what is happening? And apologies in advance if this exceeds line length for flake8

Reply:
Done!